templates/ts-apollo-mongodb-backend/src/index.ts

```diff
@@ -30,7 +32,7 @@ async function start() {
 
   const apolloServer = new ApolloServer({
     typeDefs,
-    resolvers,
+    resolvers: [resolvers, customResolvers],
```

Comment:
It would be weird to call your resolvers `customResolvers` in a regular API. We should maybe change `resolvers` to `graphbackResolvers` or call `customResolvers` `noteResolvers`

Reply:
yep!